services/response_code.proto

```diff
@@ -1437,4 +1437,10 @@ enum ResponseCodeEnum {
    * grpc_certificate_hash is invalid
    */
   INVALID_GRPC_CERTIFICATE = 345;
+
+  /**
+   * The transaction attempted to use duplicate `TokenReference`.<br/>
+   * This affects `TokenReject` attempting to reject same token reference more than once.
+   */
+  TOKEN_REFERENCE_REPEATED = 346;
```

Comment:
@Neeharika-Sompalli suggested [here](https://github.com/hashgraph/hedera-services/pull/13117#discussion_r1598711139) we could also add INVALID_OWNER_ID as status.

Maybe we could consider such a status, since for approve allowance we return INVALID_ALLOWANCE_SPENDER_ID instead of INVALID_ACCOUNT_ID

What are your thoughts on that ?

cc: @Nana-EC @MiroslavGatsanoga

Reply:
I was thinking we can use the existing `INVALID_ACCOUNT_ID` for `TokenReject` because we have only one field (`owner`) of type `AccountID` i.e. it's not ambiguous.

Reply:
@MiroslavGatsanoga I like Neeha's suggestion as it's more clear on reading and to someone who's unfamiliar with HAPI it immediately says what the issue is

Reply:
Added.